vtex-apps/store-components
# Feat/show label for variation value

#### What problem is this solving?

I would like to see the label with the variation value. Like this:

![Screenshot from 2021-01-12 16-46-34](https://user-images.githubusercontent.com/32344098/104364735-cb550700-54f5-11eb-9651-0acc7ba9f96c.png)


#### How to test it?

1. Access this workspace.
2. Click on "Iniciar sesión o registrate"
3. Click on "Crear una cuenta"
4. Register yourself on the store
5. Click on "Iniciar sesión o registrate"
6. Click on "Iniciar sesión"
7. Click on "Recibir código de acceso por e-mail"
8. Fill the input with your email.
9. Click on "Enviar"
10. Check your e-mail to get the code.
11. Fill the input with the received code.
12. Click on "Confirmar"
13. Access a product page that the product has more the one sku variation (e.g. [link to a product](https://usic144--unileverb2b.myvtex.com/)).
14. See that the label "caja" is shown with the product quantity.

[https://usic144--unileverb2b.myvtex.com/](https://usic144--unileverb2b.myvtex.com/)

#### Screenshots or example usage:

Before:
![Screenshot from 2021-01-12 17-09-29](https://user-images.githubusercontent.com/32344098/104367015-f55bf880-54f8-11eb-8213-30decd15522d.png)

After:
![Screenshot from 2021-01-12 17-10-10](https://user-images.githubusercontent.com/32344098/104367066-0b69b900-54f9-11eb-9e56-572c2018b576.png)
